src/lib/eth-calls.ts

```diff
@@ -22,4 +22,23 @@ async function registerTable(
   return await tx.wait();
 }
 
-export { registerTable };
+async function runSql(
```

Comment:
This PR only covers the direct to smart contract call for runSQL.  If all the patterns here make sense and work well, I'm thinking we create another PR for the other smart contract methods that includes implementation for #67

Reply:
👍

Reply:
Yeh I think we need these in ASAP. Let's get this merged pretty soon so we can get those in subsequently.